osama-breakout-room/Shirin Muataz/Style.css

```diff
@@ -0,0 +1,148 @@
+* {
+  padding: 0px;
+  margin: 0px;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  height: 60px;
+  box-shadow: 0px 0px 5px 0px grey;
+  padding: 6px;
+}
+
+.wrapper {
+  width: 800px;
+  margin: 0 auto;
+}
+
+.logo {
+  float: left;
+}
+
+.logo img {
+  width: 50px;
+  height: auto;
+}
+
+nav {
+  float: right;
+}
+
+.btn {
+  display: inline-block;
+  border-radius: 12px;
+  padding: 6px 8px;
+  margin: 6px;
+  text-align: center;
+  font-size: 14px;
+}
+
+a.btn {
+  text-decoration: none;
+}
+
+.btn-bordered {
+  border: 1px solid #1da1f2;
+  color: #1da1f2;
+}
+
+.btn-blue {
+  border: 1px solid #1da1f2;
+  background-color: #1da1f2;
+  color: white;
+}
+
+.profile-cover-container {
+  width: 100%;
+  height: 300px;
+}
+
+.profile-image-container {
```

Comment:
I love that you've chosen a consistent naming convention. 💪 This makes it easier to work with team members!